app/code/Magento/Inventory/Setup/InstallData.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Inventory\Setup;
+
+use Magento\Framework\Setup\InstallDataInterface;
+use Magento\Framework\Setup\ModuleDataSetupInterface;
+use Magento\Framework\Setup\ModuleContextInterface;
+use Magento\InventoryApi\Api\Data\SourceInterface;
+use Magento\InventoryApi\Api\SourceRepositoryInterface;
+use Magento\Framework\Api\DataObjectHelper;
+
+/**
+ * Class InstallData
+ */
+class InstallData implements InstallDataInterface
+{
+    /**
+     * @var SourceRepositoryInterface $stockRepository
+     */
+    private $sourceRepository;
+
+    /**
+     * @var SourceInterface
+     */
+    private $source;
+
+    /**
+     * @var DataObjectHelper
+     */
+    private $dataObjectHelper;
+
+    /**
+     * @param SourceRepositoryInterface $sourceRepository
+     * @param SourceInterface $source
+     * @param DataObjectHelper $dataObjectHelper
+     */
+    public function __construct(
+        SourceRepositoryInterface $sourceRepository,
+        SourceInterface $source,
+        DataObjectHelper $dataObjectHelper
+    ) {
+        $this->sourceRepository = $sourceRepository;
+        $this->source = $source;
+        $this->dataObjectHelper = $dataObjectHelper;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function install(ModuleDataSetupInterface $setup, ModuleContextInterface $context)
+    {
+        $this->addDefaultSource();
+    }
+
+    /**
+     * Add default source
+     *
+     * @return void
+     */
+    private function addDefaultSource()
+    {
+        $data = [
+            SourceInterface::SOURCE_ID => 1,
```

Comment:
I propose to leave it as-is now.

Because we need a decision from PO @mbrinton01  here, regarding Default Source data. And from where to take it. 
I suppose it could be some  XML  configuration with an ability to overwrite it from another module. 
But what the data would be stored there?